pkg/asset/installconfig/powervs/client.go

```diff
@@ -516,3 +517,21 @@ func (c *Client) GetAuthenticatorAPIKeyDetails(ctx context.Context) (*iamidentit
 func (c *Client) GetAPIKey() string {
 	return c.APIKey
 }
+
+// GetVPCs gets all VPCs in a region
+func (c *Client) GetVPCs(ctx context.Context, region string) ([]vpcv1.VPC, error) {
+	ctx, cancel := context.WithTimeout(ctx, 1*time.Minute)
+	defer cancel()
+
+	err := c.SetVPCServiceURLForRegion(ctx, region)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to set vpc api service url")
+	}
+
+	allVPCs := []vpcv1.VPC{}
+	if vpcs, _, err := c.vpcAPI.ListVpcs(c.vpcAPI.NewListVpcsOptions()); err != nil {
+		return nil, err
+	}
+	allVPCs = append(allVPCs, vpcs.Vpcs...)
+	return allVPCs, nil
```

Comment:
pkg/asset/installconfig/powervs/client.go:532:5: vpcs declared but not used
pkg/asset/installconfig/powervs/client.go:535:28: undefined: vpcs 

`vpcs` does not live beyound the scope of `if`.

```suggestion
	vpcs, _, err := c.vpcAPI.ListVpcs(c.vpcAPI.NewListVpcsOptions())
	if err != nil {
		return nil, err
	}
	return vpcs.Vpcs, nil
```

Reply:
Whoa, sorry about that. I thought I had force-pushed my latest commit, but apparently not. With that now pushed, it passes `go-lint`, `go-fmt`, and `go-vet` (tested locally). Let's see...